### DISCUSSION TITLE
Proxy Settings

### DISCUSSION THREAD

**Discussion by wardellcastles** [Genymobile/feedback#43] [Genymotion Desktop]
It would be nice to be able to set Proxy back to **None** after it has been set to **Manual**.     
Thank you.

**Comment by agavignet** [comment]
Hi @wardellcastles

Are you talking about the proxy that you can set in Desktop or the one in Android virtual device?

Thanks

**Comment by wardellcastles** [comment]
Good question!

 

I am talking about the setting in the Android virtual device.

 

Wardell

 

From: Adrien Gavignet ***@***.***> 
Sent: Thursday, January 19, 2023 11:55 AM
To: Genymobile/feedback ***@***.***>
Cc: wardellcastles ***@***.***>; Mention
***@***.***>
Subject: Re: [Genymobile/feedback] Proxy Settings (Discussion #43)

 

Hi @wardellcastles
<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com
%2Fwardellcastles&data=05%7C01%7C%7C165f859e15034e546b6708dafa3ddb9e%7C84df9
e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638097440842167518%7CUnknown%7CTWFpbGZ
sb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C30
00%7C%7C%7C&sdata=QcsCbDTjp9NucrDvDsuDTOWjidV5NjJB0RCB8CufEaE%3D&reserved=0>


Are you talking about the proxy that you can set in Desktop or the one in
Android virtual device?

Thanks

-
Reply to this email directly,
<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com
%2Forgs%2FGenymobile%2Fdiscussions%2F43%23discussioncomment-4729768&data=05%
7C01%7C%7C165f859e15034e546b6708dafa3ddb9e%7C84df9e7fe9f640afb435aaaaaaaaaaa
a%7C1%7C0%7C638097440842167518%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiL
CJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=HuTQVe6k
ZakuGRglO6%2FiZP1kwtOnvGX5ZWQcuyBFkHc%3D&reserved=0> view it on GitHub, or
<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com
%2Fnotifications%2Funsubscribe-auth%2FAHSQHL2UKCCUHH2IIHAWZBTWTFWVBANCNFSM6A
AAAAATXDTT4U&data=05%7C01%7C%7C165f859e15034e546b6708dafa3ddb9e%7C84df9e7fe9
f640afb435aaaaaaaaaaaa%7C1%7C0%7C638097440842167518%7CUnknown%7CTWFpbGZsb3d8
eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C
%7C%7C&sdata=tffytV7SSFcwsGy6fZDOVByaQVtgVNAUBErkeYo0LkU%3D&reserved=0>
unsubscribe.
You are receiving this because you were mentioned.
<https://github.com/notifications/beacon/AHSQHL2I37HJ4ZFMIJVLZCDWTFWVBA5CNFS
M6AAAAAATXDTT4WWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3T
UL5UWJTQAJAV2Q.gif> Message ID: <
***@***.***>
***@***.***>
